app/controllers/users_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class UsersController < ApplicationController
   def index
-    @users = User.order(:id).page(params[:page])
+    @users = User.order(:id).page(params[:page]).with_attached_avatar
```

Comment:
```suggestion
    @users = User.with_attached_avatar.order(:id).page(params[:page])
```

orderやpage/limitは最後の方に書くことが多いので、こういう書き方の方がよく見るかも？と思ったりしますが、特に明文化されていない慣習なのでこのままでも大丈夫です〜。

Reply:
ありがとうございます！！